### PR TITLE
Set-up-local-storage

### PR DESCRIPTION
**Pull request by Donard97**
We had pair programming and we use javascript to create a local storage on our browser . 

# General requirements 

- Make sure that there are no linter errors.
- Make sure that you used correct GitHub flow.
- Make sure that you documented your work in a professional way.

# Project requirements

**- You should implement the following interactions:**

- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

**- You should use the following data model:**

-   You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.

### CODE REVIEW

File: js/localstorage.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  localStorage.setItem('email-address', document.getElementById('email-address').value);
+  localStorage.setItem('message', document.getElementById('message').value);
+  setLocal();
+}
+
+if (localStorage.getItem('full-name')) {
+  populateForm();
+} else {
+ setLocal();
+}
```
Comment (dasileker) [inline js/localstorage.js]: **Hello your localStorage is awesome, but let's make it great**

- [ ] Your **`localStorage`** is storing every input field in a single object, but it should be one object to store all the fields  [see requirements](https://user-images.githubusercontent.com/61562626/128422302-91d3d9ef-336d-4f52-963b-5c69892ba91f.jpeg)

- [ ]  The data retrieved from the **`localStorage`** is vanishing after reloading the page status

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Donard97* 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/localstorage.js (hunk 1 above)
Comment (rloterh) [reply js/localstorage.js]: Issue resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Kossi, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾👍.
Congratulations! 🎉🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**